dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeChannel.java

```diff
@@ -126,11 +126,16 @@ public CompletableFuture<Object> request(Object request, int timeout, ExecutorSe
         if (closed) {
             throw new RemotingException(this.getLocalAddress(), null, "Failed to send request " + request + ", cause: The channel " + this + " is closed!");
         }
-        // create request.
-        Request req = new Request();
-        req.setVersion(Version.getProtocolVersion());
-        req.setTwoWay(true);
-        req.setData(request);
+        Request req;
+        if (request instanceof Request) {
+            req = (Request) request;
+        } else {
+            // create request.
+            req = new Request();
+            req.setVersion(Version.getProtocolVersion());
+            req.setTwoWay(true);
+            req.setData(request);
```

Comment:
What is the purpose of changing this?

Reply:
> What is the purpose of changing this?

In order to get the payload on the provider side, I added an attribute to the Request. The value of payload will be set in DubboInvoker.


The differences are as follows:
<img width="1222" alt="image" src="https://user-images.githubusercontent.com/29346818/218040923-9cf083dd-7d36-4a53-a10b-f26def09e242.png">

Reply:
Done.